docs/walkthroughs/local-setup.adoc

```diff
@@ -41,18 +42,18 @@ Clone this repository
 
 [source,bash]
 ----
-git clone git@github.com:feedhenry/mcp-standalone.git
+go get github.com/feedhenry/mcp-standalone
```

Comment:
You only need go installed if you are going to do development, so would rather leave it as git clone in this doc

Reply:
ok no problem, that's not clear from the original doc though, I'll update this PR to clarify it

Reply:
:+1:  on updating the doc to make it clearer